server/applicationPattern/applicationPattern/onfModel/services/ForwardingConstructAutomationServices.test.js

```diff
@@ -0,0 +1,439 @@
+      'client-ltp': [
+        'ro-2-0-1-http-c-or-1-0-0-000'
+      ],
+      'server-ltp': [],
+      'layer-protocol': [
+        {
+          'local-id': '0',
+          'layer-protocol-name': 'tcp-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_TCP_LAYER',
+          'tcp-client-interface-1-0:tcp-client-interface-pac': {
+            'tcp-client-interface-configuration': {
+              'remote-protocol': 'tcp-client-interface-1-0:PROTOCOL_TYPE_HTTP',
+              'remote-address': {
+                'ip-address': {
+                  'ipv-4-address': '1.1.3.1'
+                }
+              },
+              'remote-port': 3001
+            }
+          }
+        }
+      ]
+    },
+    context: undefined
+  }
+];
+
+const forwardingAutomationInputListforSubscription = [
+  {
+    forwardingName: 'OperationUpdateBroadcast',
+    attributeList: {
+      'application-name': 'RegistryOffice',
+      'release-number': '2.0.1',
+      'old-operation-name': '/v1/register-application',
+      'new-operation-name': '/v2/register-application'
+    },
+    context: undefined
+  }
+];
+
+const ltpsForSubscription = [
+  {
+    "uuid": "ro-2-0-1-op-s-is-011",
+    "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SOURCE",
+    "client-ltp": [
+    ],
+    "server-ltp": [
+      "ro-2-0-1-http-s-000"
+    ],
+    "layer-protocol": [
+      {
+        "local-id": "0",
+        "layer-protocol-name": "operation-server-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_OPERATION_LAYER",
+        "operation-server-interface-1-0:operation-server-interface-pac": {
+          "operation-server-interface-capability": {
+            "operation-name": "/v1/relay-operation-update"
+          },
+          "operation-server-interface-configuration": {
+            "life-cycle-state": "operation-server-interface-1-0:LIFE_CYCLE_STATE_TYPE_EXPERIMENTAL",
+            "operation-key": "Operation key not yet provided."
+          }
+        }
+      }
+    ]
+  }
+]
+
+const fcForSubscription = {
+  "uuid": "ro-2-0-1-op-fc-is-010",
+  "name": [
+    {
+      "value-name": "ForwardingKind",
+      "value": "core-model-1-4:FORWARDING_KIND_TYPE_SUBSCRIPTION"
+    },
+    {
+      "value-name": "ForwardingName",
+      "value": "OperationUpdateBroadcast"
+    }
+  ],
+  "fc-port": [
+    {
+      "local-id": "000",
+      "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_MANAGEMENT",
+      "logical-termination-point": "ro-2-0-1-op-s-is-003"
+    },
+    {
+      "local-id": "001",
+      "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_MANAGEMENT",
+      "logical-termination-point": "ro-2-0-1-op-s-is-002"
+    },
+    {
+      "local-id": "100",
+      "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_INPUT",
+      "logical-termination-point": "ro-2-0-1-op-s-is-011"
+    }
+  ]
+};
+
+describe("configureForwardingConstructAsync", () => {
+  test("automateForwardingConstructAsync - failed to get LogicalTerminationPointListAsync", async () => {
```

Comment:
This test and "automateForwardingConstructAsync - failed to get LogicalTerminationPointListAsync when FC Port is subscription" are identical. It does not matter that they have different attributeList, because the mocked error is thrown before the decision "isOperationOfFcPortType" is even made.